requirements/prod.txt

```diff
@@ -11,7 +11,6 @@ jedi==0.18.2
 boto==2.49.0
 boto3==1.19.4
 botocore==1.22.9
-cchardet==2.1.7
```

Comment:
cchardet is a faster version of chardet. We don't often need to detect charset on the fly and if we need to we can just use chardet. cchardet Requires a C++ compiler and it used to be an optional dependency of requests. It is abandoned - see the [drama here](https://github.com/PyYoshi/cChardet/issues/77) and everybody moved away from it.